hubblestack/utils/signing.py

```diff
@@ -63,39 +61,47 @@
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
 
-MANIFEST_RE = re.compile(r'^\s*(?P<digest>[0-9a-fA-F]+)\s+(?P<fname>.+)$')
+MANIFEST_RE = re.compile(r"^\s*(?P<digest>[0-9a-fA-F]+)\s+(?P<fname>.+)$")
 log = logging.getLogger(__name__)
 
-def _our_byte_string(x, charmap='utf-8'):
```

Comment:
I'm not sure we want to drop the charmap argument here.   I understand that it is "unused", but may make for a runtime error if we do not include it.

Reply:
Maybe add this: 
# pylint: disable=unused-argument